python/q617_testing_script.py

```diff
@@ -119,6 +25,23 @@ def cltv_passes_all_setup(lar_data=None, ts_data=None, row_count=None):
                 (lar_df['cltv'] != 'Exempt') &
                 (lar_df['cltv'] != 'NA')]
 
-#Initiates the function for creating Q617 test files. 
-new_lar_df = cltv_passes_all_setup(lar_data = lar_df, ts_data=ts_df, row_count=100)
+#Creates a five line file for each line in the row by row modification yaml.
+lar_df, ts_df = new_lar_rows(row_count=5, lar_df=lar_df, ts_df=ts_df)
```

Comment:
Is the row_count (rows created) something that might be useful to separate out? The comment implies that it is always 5, but 5 could be (maybe should be?) user defined.

Do you see any use cases for letting the user change this value?